src/ApiMethodDocumentation.js

```diff
@@ -413,8 +450,20 @@ export class ApiMethodDocumentation extends AmfHelperMixin(LitElement) {
     this.returns = this._computeReturns(method);
     if (this._isAsyncAPI(this.amf)) {
       this._overwriteExpects();
+
+      // find security from all servers for this endpoint
+      this.serversSecurity = this.asyncSecurityServers;
+
+      // security that is defined by operation
+      const methodSecurity = this._computeSecurity(method);
+
+      // method security that is defined by operation and is not defined by servers
+      this.methodSecurity = this._computeAsyncSecurityMethod(methodSecurity,this.serversSecurity);
+      this.security = this.methodSecurity.length>0?this.methodSecurity : this.serversSecurity;
```

Comment:
`this.methodSecurity.length>0?this.methodSecurity` What about adding some space `this.methodSecurity.length > 0 ? this.methodSecurity`